autoPyTorch/pipeline/components/training/trainer/RowCutMixTrainer.py

```diff
@@ -0,0 +1,60 @@
+import typing
+
+import numpy as np
+
+import torch
+
+from autoPyTorch.pipeline.components.training.trainer.base_trainer import BaseTrainerComponent
+from autoPyTorch.pipeline.components.training.trainer.mixup_utils import MixUp
+
+
+class RowCutMixTrainer(MixUp, BaseTrainerComponent):
+
+    def data_preparation(self, X: np.ndarray, y: np.ndarray,
+                         ) -> typing.Tuple[np.ndarray, typing.Dict[str, np.ndarray]]:
+        """
```

Comment:
Please provide the source of this method.

Reply:
This would be the source: https://arxiv.org/pdf/1905.04899.pdf

Modified for tabular datasets since there is no more importance related to the local placement of features.